### PR TITLE
version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -83,34 +40,29 @@ then
...
-ynh_script_progression --message="Adding a configuration file..." --weight=1
+#ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-ynh_add_config --template="../conf/config.json" --destination="$final_path/config.json"
+#ynh_add_config --template="../conf/config.json" --destination="$install_dir/config.json"
 
-chmod -R o-rwx "$final_path"
-chown -R $app:www-data "$final_path"
+#chmod 400 "$install_dir/config.json"
+#chown $app:$app "$install_dir/config.json"
```
Comment (Josue-T) [inline scripts/upgrade:56]: What is this ?

**Review by Josue-T — COMMENTED** [review]
Look me good except some small things :+1:

**Pull request merged**